### ISSUE TITLE
"Wait" Block Broken in Existing Project before July 1, 2020

### ISSUE THREAD

**Issue by MrMazzone** [thunkable/thunkable-issues#581]
### Platform
Thunkable X (iOS, Android, and Web)

### Issue/Request Origin
This issue started with the most recent release (July 1, 2020).  Was working previously. 

### Steps to Reproduce
1. Open project that used "wait" block
2. Attempt to trigger event that contains "wait" block

### Expected Behavior
The code in event with "wait" block should run without issue.

### Actual Behavior
Nothing happens when any event with "wait" block is triggered. None of the code runs in the event.  Error occurs immediately.

On Web, I was able to see this error.
![image](https://user-images.githubusercontent.com/40954645/86424887-b4fe0100-bcb1-11ea-87f7-dc0fc41b9050.png)

I tried to recreate the issue in a new project, but I cannot as it seems to not be an issue in new projects.  I am thinking that there must have been an update to the "wait" block and it is not updating in existing projects.

**Comment by josmas** [comment]
Hey @MrMazzone thanks for the report. Have you tried pulling out a new wait block and replacing the existing one? This is not ideal as a fix, but would be good to know if regenerating the Javascript from the blocks fixes the issue.

**Comment by MrMazzone** [comment]
@josmas Yes I did.  I attempted to rebuild entire events to isolate the bug.  The wait block throws the error every time in the project, new ones and old ones.  I also duplicated the project to see if it would regenerate the Javascript and it did not. 

I don't mind rebuilding all of the code, it is the UI that I just don't want to recreate.  

Of course, yesterday was the day this app was being uploaded to App Store and Play Store...  Any help is much appreciated.  Maybe there is a way for me to share the project without it being public?
